packages/client/src/clients/guide/client.ts

```diff
@@ -989,15 +1019,11 @@ export class KnockGuideClient {
     const newDebugParams = detectDebugParams();
     this.setLocation(href, { debug: newDebugParams });
```

Comment:
In setLocation(), maybe we should also clear out `previewGuides` if `preivew_session_id` is being unset?